frontend/src/component/insights/InsightsCharts.tsx

```diff
@@ -78,6 +84,16 @@ export const InsightsCharts: VFC<IChartsProps> = ({
     const showAllProjects = projects[0] === allOption.id;
     const isOneProjectSelected = projects.length === 1;
 
+    function getFlagsPerUser(
+        flags: InstanceInsightsSchemaFlags,
+        users: InstanceInsightsSchemaUsers,
+    ) {
+        const flagsPerUserCalculation = flags.total / users.total;
```

Comment:
This function can receive more primitive type, and thus be simplified. I would pass `flags.total` and `users.total` instead of entire objects. Also `formatFlagsPerUser` would be more descriptive to me.

Reply:
Not sure I understand what you mean here

Reply:
This function doesn't need to have complex arguments. It can probably work with `? number | null` arguments instead, and be called as `formatFlagsPerUser(flags.total, users.total)